src/plugins/telemetryTable/components/TableComponent.vue

```diff
@@ -478,7 +478,7 @@ export default {
     }
   },
   created() {
-    this.filterChanged = _.debounce(this.filterChanged, 500);
+    this.filterTelemetry = _.debounce(this.filterTelemetry, 500);
```

Comment:
This would make a pretty neat composable if you're looking to get your hands dirty. Something like `useDebounceFn` maybe

Reply:
Noted!